packages/fiori/src/NotificationListInternal.ts

```diff
@@ -36,6 +38,42 @@ class NotificationListInternal extends List {
 
 		return items;
 	}
+
+	_onkeydown(e: KeyboardEvent) {
+		super._onkeydown(e);
+		this.focusSameItemOnNextRow(e);
+	}
+
+	focusSameItemOnNextRow(e: KeyboardEvent) {
+		const target = e.target as HTMLElement;
+		const shadowTarget = target.shadowRoot!.activeElement as HTMLElement;
+		if (!target || target.hasAttribute("ui5-menu-item")) {
+			return;
+		}
+
+		const isFocusWithin = target.matches(":focus-within");
+		if (!isFocusWithin || (!isUp(e) && !isDown(e))) {
+			return;
+		}
+
+		e.preventDefault();
+		e.stopImmediatePropagation();
+
+		const navItems = this.getEnabledItems();
+		// @ts-expect-error TOFIX strictEvents
+		const index = navItems.indexOf(target) + (isUp(e) ? -1 : 1);
+		const nextItem = navItems[index] as NotificationListItemBase;
+		if (!nextItem) {
+			return;
+		}
+
+		const sameItemOnNextRow = nextItem.getHeaderDomRef()!.querySelector(`.${shadowTarget.className}`) as HTMLElement;
+		if (sameItemOnNextRow && sameItemOnNextRow.offsetParent) {
```

Comment:
It is wrong to use cast to HTMLElement if it requires check for existence on next line. querySelector is a generic function and if html is passed to generic it will return correct definition.